### ISSUE TITLE
Android App crashes 

### ISSUE THREAD

**Issue by Abgaryan** [loup-v/geolocation#17]
``E/AndroidRuntime( 9484): java.lang.IllegalArgumentException: Can only use lower 16 bits for requestCode
E/AndroidRuntime( 9484): 	at android.support.v4.app.BaseFragmentActivityApi14.checkForValidRequestCode(BaseFragmentActivityApi14.java:79)
E/AndroidRuntime( 9484): 	at android.support.v4.app.FragmentActivity.validateRequestPermissionsRequestCode(FragmentActivity.java:765)
E/AndroidRuntime( 9484): 	at android.support.v4.app.ActivityCompat.requestPermissions(ActivityCompat.java:505)
E/AndroidRuntime( 9484): 	at io.intheloup.geolocation.location.LocationClient.requestPermission(LocationClient.kt:288)
E/AndroidRuntime( 9484): 	at io.intheloup.geolocation.location.LocationClient.validateServiceStatus(LocationClient.kt:245)
E/AndroidRuntime( 9484): 	at io.intheloup.geolocation.location.LocationClient.requestLocationPermission(LocationClient.kt:71)
E/AndroidRuntime( 9484): 	at io.intheloup.geolocation.LocationChannel$requestLocationPermission$1.doResume(LocationChannel.kt:31)
E/AndroidRuntime( 9484): 	at kotlin.coroutines.experimental.jvm.internal.CoroutineImpl.resume(CoroutineImpl.kt:54)
E/AndroidRuntime( 9484): 	at kotlinx.coroutines.experimental.DispatchedTask$DefaultImpls.run(Dispatched.kt:161)
E/AndroidRuntime( 9484): 	at kotlinx.coroutines.experimental.DispatchedContinuation.run(Dispatched.kt:25)
E/AndroidRuntime( 9484): 	at android.os.Handler.handleCallback(Handler.java:739)
E/AndroidRuntime( 9484): 	at android.os.Handler.dispatchMessage(Handler.java:95)
E/AndroidRuntime( 9484): 	at android.os.Looper.loop(Looper.java:158)
E/AndroidRuntime( 9484): 	at android.app.ActivityThread.main(ActivityThread.java:7229)
E/AndroidRuntime( 9484): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime( 9484): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
E/AndroidRuntime( 9484): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)``

**Comment by Abgaryan** [comment]
@lukaspili you an close the issue solved in https://github.com/loup-v/geolocation/pull/18

**Comment by Asoseil** [comment]
Hi, my app crash if i use geolocation with firebase. Separately works fine, i tried with a blank app and when i add firebase with geolocation crash during the building phase. Could be a dependencies problem? I'm using latest version of flutter, firebase and geolocation

**Comment by renecura** [comment]
Same problem here with firebase (cloud_firestore).
Only adding the dependencies in pubspec.yaml breaks the app.

```
dependencies:
  cloud_firestore: ^0.7.3
  geolocation: ^0.2.1
  flutter:
    sdk: flutter
```

**Comment by lukaspili** [comment]
Sorry for the delay. It's fixed in the latest release 1.0.0.
